stix_shifter_modules/aws_cloud_watch_logs/stix_translation/json/to_stix_map.json

```diff
@@ -598,46 +598,45 @@
         }
       ],
       "detail_resource_instanceDetails_imageId":{
-        "key": "x_aws_instance.image_id",
-        "cybox": false
+        "key": "x-aws-instance.image_id",
+        "object": "x_aws_instance"
       },
       "detail_resource_instanceDetails_instanceId": {
-        "key": "x_aws_instance.instance_id",
-        "cybox": false
+        "key": "x-aws-instance.instance_id",
+        "object": "x_aws_instance"
       },
       "detail_resource_instanceDetails_availabilityZone": {
-        "key": "x_aws_instance.availability_zone",
-        "cybox": false
+        "key": "x-aws-instance.availability_zone",
+        "object": "x_aws_instance"
       }
     },
     "@timestamp": {
-        "key": "x_aws_guardduty_finding.timestamp",
-        "cybox": false,
-        "transformer": "AwsToTimestamp"
+        "key": "x-aws-guardduty-finding.timestamp",
```

Comment:
I think this can be removed since `@timestamp` also maps to `first_observed` and `last_observed`.